src/navigations/root-stack.js

```diff
@@ -186,7 +186,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="OtherProfile"
-          component={withKeyboardWrapper(OtherProfile)}
+          component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change wraps the `OtherProfile` component with `withSafeAreaView`. This is a good practice to ensure that the component doesn't overlap with system UI elements on devices with notches or rounded corners. However, please make sure that this doesn't introduce any unwanted padding or spacing issues in the layout of the `OtherProfile` screen.
```diff
-          component={withKeyboardWrapper(OtherProfile)}
+          component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->